### ISSUE TITLE
Error training Unigram Tagger with Indian Corpus Data

### ISSUE THREAD

**Issue by djokester** [nltk/nltk#1653]
```
sents = nltk.corpus.indian.tagged_sents()
# 1280 is the index where the Bengali or Bangla corpus ends. 
>>> train = sents[0:int(1280)*0.9)]
>>> test = sents[int(l1280*0.9):]
>>> ug = nltk.UnigramTagger(train)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/djokester/anaconda3/lib/python3.5/site-packages/nltk/tag/sequential.py", line 340, in __init__
    backoff, cutoff, verbose)
  File "/home/djokester/anaconda3/lib/python3.5/site-packages/nltk/tag/sequential.py", line 287, in __init__
    self._train(train, cutoff, verbose)
  File "/home/djokester/anaconda3/lib/python3.5/site-packages/nltk/tag/sequential.py", line 178, in _train
    tokens, tags = zip(*sentence)
ValueError: not enough values to unpack (expected 2, got 0)
```
Usually this is the general pathway we follow while training any Ngram tagger with Brown ortreebank corpus. But this doesn't work with the Indian corpus.Is there an error on my part or is this a bug?

**Comment by alvations** [comment]
Thanks @djokester for catching the error! 

This looks like a similar problem we had with the hindi portion of the corpus where there's an empty sentence c.f. #661

```
>>> from nltk.corpus import indian
>>> bengali_sents = indian.tagged_sents('bangla.pos')
>>> [i for i, sent in enumerate(bengali_sents) if not sent]
[297, 376, 824]

>>> marathi_sents = indian.tagged_sents('marathi.pos')
>>> [i for i, sent in enumerate(marathi_sents) if not sent]
[]
>>> telugu_sents = indian.tagged_sents('telugu.pos')
>>> [i for i, sent in enumerate(telugu_sents) if not sent]
[]
>>> hindi_sents = indian.tagged_sents('hindi.pos')
>>> [i for i, sent in enumerate(hindi_sents) if not sent]
[]
```

The solution would be to remove the empty lines from `nltk_data`. 

These entries from `bangla.pos` in `nltk_data/corpora/indian.zip` is causing the error:

```
<Sentence id=298></Sentence>
<Sentence id=377></Sentence>
<Sentence id=825></Sentence>
```

----

Meanwhile, to train the tagger, @djokester you can do this:

```python
>>> from nltk.corpus import indian
>>> from nltk.tag import UnigramTagger

>>> bengali_sents = indian.tagged_sents('bangla.pos')
>>> bengali_sents = [sent for sent in bengali_sents if sent]
>>> len(bengali_sents)
899
>>> bengali_sents = [sent for sent in bengali_sents if sent]
>>> len(bengali_sents)
896

>>> split_pt = int(len(bengali_sents) * 0.9)
>>> train_set = bengali_sents[:split_pt]
>>> test_set = bengali_sents[split_pt:]

>>> ut = UnigramTagger(train_set)
>>> ut.tag(test_set[0])
[((u'\u099a\u09bf\u09a0\u09bf\u099f\u09be\u09a4\u09c7', u'NN'), None), ((u'\u0985\u09a8\u09c7\u0995', u'QF'), None), ((u'\u09ad\u09c1\u09b2\u09ad\u09cd\u09b0\u09be\u09a8\u09cd\u09a4\u09bf', u'NN'), None), ((u'\u09a5\u09be\u0995\u09b2\u09c7\u0993', u'VM'), None), ((u'\u098f\u09b0', u'PRP'), None), ((u'\u09aa\u09cd\u09b0\u09a4\u09bf\u099f\u09bf', u'QF'), None), ((u'\u099b\u09a4\u09cd\u09b0\u09c7', u'NN'), None), ((u'\u09af\u09c7\u09a8', u'RP'), None), ((u'\u09b2\u0995\u09cd\u09b7\u09cd\u09ae\u09c0\u09b0', u'NNP'), None), ((u'\u099c\u09c0\u09ac\u09a8', u'XC'), None), ((u'\u09aa\u09bf\u09aa\u09be\u09b8\u09be\u09b0', u'NN'), None), ((u'\u099b\u09ac\u09bf', u'NN'), None), ((u'\u0986\u0981\u0995\u09be', u'VM'), None), ((u'\u09b9\u09df\u09c7\u099b\u09c7', u'VAUX'), None), ((u'\u09f7', u'SYM'), None)]
```

**Comment by djokester** [comment]
So there are a few empty lines in the corpus. That is what is causing the error? 
Mind if I take it up as my first PR to NLTK?

**Comment by alvations** [comment]
> Mind if I take it up as my first PR to NLTK?

Contribution is always welcomed!! =)

But do note that the fix would be in the [`nltk_data`](https://github.com/nltk/nltk_data) repo instead of nltk. So beyond the contribution guide: https://github.com/nltk/nltk/blob/develop/CONTRIBUTING.md , do also look at https://github.com/nltk/nltk/wiki/Adding-a-Corpus

**Comment by alvations** [comment]
Fixed on nltk/nltk_data#67
